### PR TITLE
Building a Toxicity Classifier using Python

### LINKED ISSUE #4742: [Machine Learning] Building a Toxicity Classifier with Python Lightning and Transformers
#### Proposed title of article
Your title should be descriptive of the article/tutorial. Be Specific. 

Building a Toxicity Classifier with Python Lightning and Transformers

#### Proposed article introduction
*We're looking for the first 2-3 paragraphs of your article that appropriately summarize what your article will be about.*
We will use this writing snippet to help us assess the overall quality before approval so ensure it has been proofread, spellchecked and grammarchecked.

The toxicity classifier model detects whether text contains toxic content such as threatening language, insults, obscenities, identity-based hate, or sexually explicit language. This tutorial will use the toxicity pre-trained model to recognize sentences that are toxic. We will also integrate this model into the [Gradio app](https://gradio.app/) to build an interactive application that will allow the user to enter a sentence and the model will in turn give an output if the sentence is toxic or not.

#### Key takeaways 
What are the 3-5 most important things the reader should understand or be able to do after reading this article?
Use this area to get your ideas down on the bulk of your article or tutorial.

By the end of this tutorial, a reader should be able to:
1. Learn how to install and import the necessary dependencies.
2. Learn how to import the toxic classifier model into our Jupyter notebook/Google Colab.
3. Build the toxic classifier with PyTorch and integrate it into the Gradio app.

#### References
Please list links to any published content/research that you intend to use to support/guide this article.
- [Dataset](https://www.kaggle.com/c/jigsaw-multilingual-toxic-comment-classification/data)
- [Toxicity classifier model](https://github.com/tensorflow/tfjs-models/tree/master/toxicity)

#### Article quality
How is your article unique? Tell us what makes your approach different from similar articles that have published on the same topic?
Is yours more in-depth? Does it cover additional topics? Do you provide handy tips or anecdotal advice?

### PR DESCRIPTION
**Pull request by lcheptoo**
This closes #4742

### CODE REVIEW

**Comment by WanjaMIKE** [comment]
Plagiarism check done.

File: content/articles/building-a-toxicity-classifier/index.md [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+predictor = Detoxify('multilingual')
+```
+
+Let's try and perform prediction on a simple sentence to establish whether it's toxic or not.
+
+```python
+predictor.predict('Why are you so fat?')
+```
+In some modern cultures, calling someone `fat` isn't such a big issue. But in others, people mind find it offensive. Let's see what the model thinks.
+
```
Comment (Bayler) [inline content/articles/building-a-toxicity-classifier/index.md:71]:
```suggestion
In some modern cultures, calling someone `fat` isn't such a big issue. But in other cultures, they might find it offensive. Let's see what the model thinks.
```

File: content/articles/building-a-toxicity-classifier/index.md [hunk 2]
```diff
@@ -0,0 +1,195 @@
...
+- [Deploying the model as an application using Gradio](#deploying-the-model-as-an-application-using-gradio)
+- [Wrapping up](wrapping-up)
+
+### The toxicity classifier
+Toxicity classier can classify whether a particular text or sentence is toxic or not. Cyberbullying has been an issue ever since the rise of the social media era. Many people have experienced cyberbullying in one way or another. On social media sites, there are a lot of comments that people make, and most of them have a significant amount of negativity and toxicity in them.
+
+Major technology companies have turned to machine learning to help with the automated classification of toxic content to combat toxic content and online abuse. Indeed, many of these automated systems are deployed today. For example, this has been seen on Twitter, where the company has automatically flagged more than half of all tweets that violate its rules. Just recently, in January 2021, Donald Trump's Twitter account was [permanently suspended](https://blog.twitter.com/en_us/topics/company/2020/suspension) as his account was deemed to cause incitement of violence. A machine learning system aided this.
```
Comment (Bayler) [inline content/articles/building-a-toxicity-classifier/index.md:37]:
```suggestion
Major technology companies have turned to machine learning to help with the automated classification of toxic content to combat toxic content and online abuse. Indeed, many of these automated systems are deployed today. For example, this has been seen on Twitter, where the company has automatically flagged more than half of all tweets that violate its rules. Just recently, in January 2021, Donald Trump's Twitter account was [permanently suspended](https://blog.twitter.com/en_us/topics/company/2020/suspension) as his account was deemed to cause incitement of violence. This automation is performed by a machine learning system.  
```

**Comment by Bayler** [comment]
This was very educative. I really learned a lot. Your tutorial is now ready for final review.

**Comment by hectorkambow** [comment]
jumping on this one

**Comment by hectorkambow** [comment]
Nice work 👍  

Very easy to follow along 🚀

**Pull request merged**